site/zenodo_rdm/assets/semantic-ui/js/zenodo_rdm/src/support/components/SupportForm.js

```diff
@@ -16,7 +16,6 @@ import FileUploader from "./FileUploader";
 
 const requestConfig = {
   headers: {
-    "Accept": "application/json",
     "Content-Type": "multipart/form-data",
```

Comment:
I think it needs to specify the accept header, it looks to me that here https://github.com/zenodo/zenodo-rdm/pull/684/files#diff-e1f7092b13856cf547fcc0930acf3af64025c718e6860dd6e619c528302d60cbL85 headers are overridden, not updated